### PR TITLE
Improved eventing getting-started guide

### PR DESCRIPTION
**Pull request by mfaizanse**
**Description**

Changes proposed in this pull request:

- Addressed review comments from PR: https://github.com/kyma-project/kyma/pull/14703

**Related issue(s)**

- https://github.com/kyma-project/kyma/issues/14193

### CODE REVIEW

**Review by k15r — APPROVED** [review]

File: docs/02-get-started/04-trigger-workload-with-event.md [hunk 1]
```diff
@@ -94,8 +94,8 @@ kubectl get functions -n default lastorder
 
 ## Create a Subscription
 
-Next, to subscribe to an event so that we can actually listen for it, we need a [Subscription](../05-technical-reference/00-custom-resources/evnt-01-subscription.md) custom resource. We're going to be listening for an event of type `order.received.v1`. 
-All the events published against this event type will be forwarded to the `Sink` (i.e. HTTP endpoint) defined in the Subscription's Spec.
+Next, to subscribe to an event, we need a [Subscription](../05-technical-reference/00-custom-resources/evnt-01-subscription.md) custom resource. We're going to subscribe for an event of type `order.received.v1`.
```
Comment (klaudiagrz) [inline docs/02-get-started/04-trigger-workload-with-event.md:97]:
```suggestion
Next, to subscribe to an event, we need a [Subscription](../05-technical-reference/00-custom-resources/evnt-01-subscription.md) custom resource. We're going to subscribe to an event of the type `order.received.v1`. 
```

File: docs/02-get-started/04-trigger-workload-with-event.md [hunk 2]
```diff
@@ -94,8 +94,8 @@ kubectl get functions -n default lastorder
 
 ## Create a Subscription
 
-Next, to subscribe to an event so that we can actually listen for it, we need a [Subscription](../05-technical-reference/00-custom-resources/evnt-01-subscription.md) custom resource. We're going to be listening for an event of type `order.received.v1`. 
-All the events published against this event type will be forwarded to the `Sink` (i.e. HTTP endpoint) defined in the Subscription's Spec.
+Next, to subscribe to an event, we need a [Subscription](../05-technical-reference/00-custom-resources/evnt-01-subscription.md) custom resource. We're going to subscribe for an event of type `order.received.v1`. 
+All the events published on this event type will be forwarded to the `Sink` (i.e. HTTP endpoint) defined in the Subscription's Spec.
```
Comment (klaudiagrz) [inline docs/02-get-started/04-trigger-workload-with-event.md:98]:
```suggestion
All the published events of this type are then forwarded to an HTTP endpoint called `Sink`. You can define this endpoint in the Subscription's spec.
```

File: docs/02-get-started/04-trigger-workload-with-event.md [hunk 3]
```diff
@@ -157,7 +157,7 @@ The operation was successful if the returned status says `true`.
 
 ## Trigger the workload with an event
 
-We created the `lastorder` Function, and created a Subscription for it to listen for `order.received.v1` events. Now it's time to send an event and trigger the Function. In this example, we'll port-forward the Kyma Eventing Service to localhost. 
+We created the `lastorder` Function, and subscribed for `order.received.v1` events by creating a Subscription CR. Now it's time to publish an event and trigger the Function. In this example, we'll port-forward the Kyma Eventing Service to localhost.
```
Comment (klaudiagrz) [inline docs/02-get-started/04-trigger-workload-with-event.md:160]:
```suggestion
We created the `lastorder` Function and subscribed to the `order.received.v1` event by creating a Subscription CR. Now it's time to publish your event and trigger the Function. In this example, we'll port-forward the Kyma Eventing Service to localhost. 
```

Comment (klaudiagrz) [reply docs/02-get-started/04-trigger-workload-with-event.md:160]: We're talking about one event here, is that correct?

Comment (mfaizanse) [reply docs/02-get-started/04-trigger-workload-with-event.md:160]: yes

File: docs/02-get-started/04-trigger-workload-with-event.md (hunk 1 above)
Comment (mfaizanse) [reply docs/02-get-started/04-trigger-workload-with-event.md]: I updated the line as:
```
Next, to subscribe to events, we need a [Subscription](../05-technical-reference/00-custom-resources/evnt-01-subscription.md) custom resource. We're going to subscribe to events of the type `order.received.v1`.
```

**Review by klaudiagrz — APPROVED** [review]

**Pull request merged**